### DISCUSSION TITLE
The Sidebar should take up less space when expanded

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6678] [Serenity Ideas]
The sidebar in Serenity takes up too much space when expanded. Usually, when a sidebar is expanded, the leftmost side should disappear (like in the old version of Serenity as well as most other websites). I am referring to the dark blue bar below:
![image](https://user-images.githubusercontent.com/57414714/217269285-e3b6e527-fa91-48ab-9641-358cb16ca21c.png)

This is a problem because the sidebar ends up taking up too much space on smaller screens like laptops and leaves less space to view grids.

**Comment by edwardch** [comment]
When you say the left side "should disappear" what do you mean?
If it disappears then you won't be able to click the top level menu items.

I think it would be useful to know how to minimise the size of the menu bar, though.

**Reply by ibrahimGit123** [reply]
Yes - I think the top level Menu Items should not exist. They should instead of be incorporated into the right-side Menu, as is typically the case, or they should be removed totally.

**Reply by ibrahimGit123** [reply]
In earlier versions of Serenity, the icons on the left-most menu (account icon, language) were on the top of the page instead of the side bar

**Comment by dfaruque** [comment]
People have different taste in different look and feel. No single design can satisfy everyone. The serenity team can consider adding an option for changing the layout change just like the theme change.
For example, Gmail has settings to change the layout.
![image](https://user-images.githubusercontent.com/16621563/218294943-0d4bd302-837f-4926-81d0-aab9cd2579ad.png)
